source/manual/data-gov-uk-supporting-ckan.html.md

```diff
@@ -8,43 +8,55 @@ last_reviewed_on: 2018-05-21
 review_in: 3 months
 ---
 [ckan]: https://ckan.org
-[ckanext-datagovuk]: apps/ckanext_datagovuk
-[ckan deployment]: https://github.com/alphagov/datagovuk_ckan_deployment
-[infrastructure]: https://github.com/alphagov/datagovuk_infrastructure
+[ckanext-datagovuk]: /apps/ckanext-datagovuk.html
 [ckandocs]: http://docs.ckan.org/en/ckan-2.7.3/maintaining/paster.html
 
 ## Environments
 
-We currently have one environment: development.
+There are three currently three environments for [CKAN]:
```

Comment:
"There are currently three environments for [CKAN]"

Reply:
Fixed up and rebased.